docs/pages/careers/support-agent.md

```diff
@@ -1,104 +0,0 @@
-# Customer Support Agent
```

Comment:
We try to keep all the .md file in the repository as it's simpler to revert when we open new role. Similar to how in Ashby we have open / closed / archived roles, e.g. https://app.ashbyhq.com/jobs/f1da808b-03a4-4396-a1ed-18ab3a586869/candidate-pipeline/hired. I'm restoring this in https://github.com/mui/material-ui/pull/42058. (archived -> closed).